listenbrainz/webserver/views/settings.py

```diff
@@ -354,17 +269,16 @@ def music_services_details():
     soundcloud_user = soundcloud_service.get_user(current_user.id)
     current_soundcloud_permissions = "listen" if soundcloud_user else "disable"
 
-    return render_template(
-        'user/music_services.html',
-        spotify_user=spotify_user,
-        current_spotify_permissions=current_spotify_permissions,
-        critiquebrainz_user=critiquebrainz_user,
-        current_critiquebrainz_permissions=current_critiquebrainz_permissions,
-        current_soundcloud_permissions=current_soundcloud_permissions,
-        active_settings_section="connect-services"
-    )
+    data = {
+        "current_spotify_permissions": current_spotify_permissions,
+        "current_critiquebrainz_permissions": current_critiquebrainz_permissions,
+        "current_soundcloud_permissions": current_soundcloud_permissions,
+    }
+
+    return jsonify(data)
 
 
+@settings_bp.route('/music-services/<service_name>/callback/')
```

Comment:
Since the routes have migrated, I've enabled callback URLs for both the `/profile` and `/settings` routes. Do we want to keep both of these callback URLs or just one? @MonkeyDo